core/integration/legacy_test.go

```diff
@@ -756,3 +756,78 @@ func (LegacySuite) TestContainerWithFocus(ctx context.Context, t *testctx.T) {
 		})
 	require.NoError(t, err)
 }
+
+func (LegacySuite) TestContainerAsService(ctx context.Context, t *testctx.T) {
```

Comment:
Can you add a comment like in the other tests explaining what this test is specifically looking for, and which PR changed it?

It's quite useful when readying through these later.